pages/seller/js/seller.js

```diff
@@ -73,34 +92,30 @@ let renderProducts = () =>{
     let detail = document.createElement('div');
     detail.className = 'detail';  
     productInfo.appendChild(detail);
-    // console.log(productInfo)
 
-    let names = document.createElement('span');
+    let names = document.createElement('p');
     names.className = 'name';
-    names.textContent = product.name;
+    names.textContent = product.name + ' ' + product.type;
```

Comment:
Use modern syntax for concatenation: 
names.textContent = product.name + ' ' + product.type;
names.textContent = \`${product.name} ${product.type}\`